Core/Core/Common/CommonUI/CoreWebView/Model/Features/InsertStudioOpenDetailViewButton.swift

```diff
@@ -0,0 +1,143 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2025-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+class InsertStudioOpenInDetailButtons: CoreWebViewFeature {
+
+    private let insertStyle: String = {
+        let css = """
+        p[ios-injected] {
+            text-align: center;
+        }
+
+        .open_details_button {
+            font-weight: 400;
+            font-size: 15px;
```

Comment:
The button doesn't grow with dynamic font sizes.

<img width="820" height="1180" alt="Image" src="https://github.com/user-attachments/assets/a5dcbd7f-93d6-4443-ad7b-d33522da01de" />